packages/next/server/web/spec-extension/cookies.ts

```diff
@@ -0,0 +1,116 @@
+import cookie from 'next/dist/compiled/cookie'
+import { CookieSerializeOptions } from '../types'
+
+const normalizeCookieOptions = (options: CookieSerializeOptions) => {
+  options = Object.assign({}, options)
+
+  if (options.maxAge) {
+    options.expires = new Date(Date.now() + options.maxAge)
+    options.maxAge /= 1000
```

Comment:
Do we maybe want this to require seconds from the developer?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#:~:text=not%20the%20server.-,Max%2DAge,-%3D%3Cnumber%3E%20Optional

See also: https://github.com/vercel/next.js/issues/33472

Reply:
Good point; I will update the PR!